### PR TITLE
Otaku-Streamers Presence

### PR DESCRIPTION
**Pull request by Gurrman375**
otaku-streamers.com:
image
image
image
image
image
beta.otaku-streamers.com:
image
image
image
image
image

### CODE REVIEW

File: websites/H/Hulu/dist/metadata.json [hunk 1]
```diff
@@ -1,9 +1,14 @@
...
-    "name": "Alanexei ✔",
-    "id": "163319338403627008"
+    "name": "GURRMAN375D",
+    "id": "446528923535278082"
   },
   "contributors": [
+    {
+      "name": "Alanexei ✓",
+      "id": "163319338403627008"
+    },
```
Comment (Bas950) [inline websites/H/Hulu/dist/metadata.json:11]: Did alan agree to this?

File: websites/H/Hulu/presence.ts [hunk 2]
```diff
@@ -156,7 +156,7 @@ presence.on("UpdateData", async () => {
     }
   } else if (path.includes("/watch")) {
     video = document.querySelector(".video-player");
-    details = "Viewing Watch History";
+    // details = "Viewing Watch History";
```
Comment (Bas950) [inline websites/H/Hulu/presence.ts:159]: is this correct?

File: websites/H/Hulu/dist/metadata.json (hunk 1 above)
Comment (Gurrman375) [reply websites/H/Hulu/dist/metadata.json:11]: https://github.com/PreMiD/Presences/pull/1725

File: websites/H/Hulu/presence.ts (hunk 2 above)
Comment (Gurrman375) [reply websites/H/Hulu/presence.ts]: Yeah that would break it.

**Comment by doomlerd** [comment]
post Hulu proof as well

**Comment by Gurrman375** [comment]
Here's Proof:
image
image
The issue is unless @doomlerd checks it on his end the proof shown above is meaningless.

**Pull request merged**